### PR TITLE
Default PgSQL charset to utf8

### PR DESCRIPTION
**Pull request by RomainMazB**
This would fit 99% of use case for #9 as almost all of us doesn't pay attention to the charset when creating a database or the default just fit what we need.

### CODE REVIEW

**Comment by mjauvin** [comment]
collation should only be provided for mysql, and charset should be "utf8" for both pgsql and sqlserver.

**Comment by RomainMazB** [comment]
The collation didn't create an issue, because it's not even used by the Laravel's connector:
- [MySQL uses it](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/Connectors/MySqlConnector.php#L76)
- [PgSQL doesn't](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/Connectors/PostgresConnector.php#L68)

File: public/install/api/src/Api.php [hunk 1]
```diff
@@ -714,7 +714,7 @@ protected function createCapsule(array $dbConfig)
                     'database' => $dbConfig['name'],
                     'username' => $dbConfig['user'] ?? '',
                     'password' => $dbConfig['pass'] ?? '',
-                    'charset' => 'utf8mb4',
+                    'charset' => $dbConfig['type'] === 'mysql' ? 'utf8mb4' : 'utf8',
```
Comment (bennothommo) [inline public/install/api/src/Api.php:717]:
```suggestion
                    'charset' => ($dbConfig['type'] === 'mysql') ? 'utf8mb4' : 'utf8',
```

File: public/install/api/src/Api.php [hunk 2]
```diff
@@ -714,7 +714,7 @@ protected function createCapsule(array $dbConfig)
                     'database' => $dbConfig['name'],
                     'username' => $dbConfig['user'] ?? '',
                     'password' => $dbConfig['pass'] ?? '',
-                    'charset' => 'utf8mb4',
+                    'charset' => ($dbConfig['type'] === 'mysql') ? 'utf8mb4' : 'utf8',
                     'collation' => 'utf8mb4_unicode_ci',
```
Comment (bennothommo) [inline public/install/api/src/Api.php:718]:
```suggestion
                    'collation' => ($dbConfig['type'] === 'mysql') ? 'utf8mb4_unicode_ci' : null,
```

**Comment by bennothommo** [comment]
@RomainMazB I've just adjusted this to make it explicit that the collation is only used for MySQL.

**Pull request merged**